miner/miner.go

```diff
@@ -93,3 +103,28 @@ func (mm *MinerManager) GetDealProtocolForMiner(ctx context.Context, miner addre
 	}
 	return proto, nil
 }
+
+func (mm *MinerManager) GetMinerChainInfo(ctx context.Context, maddr address.Address) (*MinerChainInfo, error) {
```

Comment:
should we cache the `miner - chainInfo` somewhere to reduce the on-chain lookups required?

Do we have an existing cache in place where we can put it?